airflow/providers/amazon/aws/operators/lambda_function.py

```diff
@@ -117,6 +144,13 @@ def execute(self, context: Context):
 
         return response.get("FunctionArn")
 
+    def execute_complete(self, context: Context, event: dict[str, Any] | None = None) -> None:
```

Comment:
```suggestion
    def execute_complete(self, context: Context, event: dict[str, Any] | None = None) -> str:
```
It's weird that static checks are not picking up this one